### DISCUSSION TITLE
[@types/react-native-share-menu] package is updated to allow multiple items

### DISCUSSION THREAD

**Discussion by sensecodevalue** [DefinitelyTyped/DefinitelyTyped#60705] [Issues with a @types package]
Hello! First of all, thank you for your contribution.

I was using the react-native-share-menu.

There was a problem with the return type of ShareMenuReactView.data().

26 days ago, there was an update in the react-native-share-menu that could share multiple items.
- https://github.com/meedan/react-native-share-menu/pull/158

Therefore, when the type was changed as follows, it could be used without error.

```Diff
diff --git a/node_modules/@types/react-native-share-menu/index.d.ts b/node_modules/@types/react-native-share-menu/index.d.ts
index f52822c..7debfa1 100755
--- a/node_modules/@types/react-native-share-menu/index.d.ts
+++ b/node_modules/@types/react-native-share-menu/index.d.ts
@@ -28,7 +28,7 @@ interface ShareMenuReactView {
     dismissExtension(error?: string): void;
     openApp(): void;
     continueInApp(extraData?: object): void;
-    data(): Promise<{mimeType: string, data: string}>;
+    data(): Promise<{data: ShareData[]}>;
 }
```

[a comment by an automated account is not shown]

**Comment by PhilBeaudoin** [comment]
@SenseCodeValue Thanks for the update. Would you be able to submit the change as a PR?

**Reply by sensecodevalue** [reply]
@PhilBeaudoin 
Thank you for your answer!
I will upload PR according to the rules asap.
